package.json

```diff
@@ -21,8 +21,8 @@
     "release": "yarn beachball publish"
   },
   "devDependencies": {
-    "beachball": "^2.1.0",
-    "lage": "0.29.0",
+    "beachball": "2.2.0",
+    "lage": "0.29.1",
```

Comment:
Did you use fixed version requirements here on purpose?

Reply:
Just precautions, I know yarn.lock works - but usually when I upgrade, I'd like to exact on these two tools.